_build_cfg.yml

```diff
@@ -192,6 +192,8 @@ Topics:
         File: persistent_storage_glusterfs
       - Name: Using OpenStack Cinder
         File: persistent_storage_cinder
+      - Name: Using Ceph RBD
+        File: persistent_storage_ceph
```

Comment:
Should the file name here be more specifically something like `persistent_storage_ceph_rbd` or `persistent_storage_rbd` since it appears `cephfs` is also a valid volume plug-in (https://docs.openshift.org/latest/rest_api/kubernetes_v1.html#v1-volume)? Even if we're not writing about it yet.